File: Application_Module/Emergency_webserver/api/barrack.py
from flask import Flask, jsonify
from app import app
import database.service as db

@app.route('/api/getBarracks/')
def getBarracksAPI():
    data = db.get(['id','longitude','latitude'],"fireworker.barracks")
    data2 = []
    for datum in data:
        data2.append({
            "m_id" : datum["id"],
            "m_longitude" : datum["longitude"],
            "m_latitude" : datum["latitude"]
        })    
    return jsonify(data2)

File: Application_Module/EmmergencyManager/src/main/java/com/Objects/Barrack.java
package com.Objects;

public class Barrack {
    private Integer m_id;
    private Double m_longitude;
    private Double m_latitude;

    public Barrack(Integer m_id, Double m_longitude, Double m_latitude) {
        this.m_id = m_id;
        this.m_longitude = m_longitude;
        this.m_latitude = m_latitude;
    }

    @Override
    public String toString() {
        return "Barrack{" +
                "m_id=" + m_id +
                ", m_longitude=" + m_longitude +
                ", m_latitude=" + m_latitude +
                '}';
    }
}
